fdbclient/ServerKnobs.cpp

```diff
@@ -739,7 +739,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( GLOBAL_TAG_THROTTLING,                               false ); if(isSimulated) GLOBAL_TAG_THROTTLING = deterministicRandom()->coinflip();
 	init( ENFORCE_TAG_THROTTLING_ON_PROXIES,   GLOBAL_TAG_THROTTLING );
 	init( GLOBAL_TAG_THROTTLING_MIN_RATE,                        1.0 );
-	init( GLOBAL_TAG_THROTTLING_FOLDING_TIME,                   10.0 );
+	init( GLOBAL_TAG_THROTTLING_FOLDING_TIME,                   60.0 );
```

Comment:
Optional: I don't see many comments in this file for how various knob values were chosen, but if it seems appropriate, it might be useful to add that info somewhere for this knob since you did some testing.